rasa/core/policies/ted_policy.py

```diff
@@ -174,10 +180,10 @@ class TEDPolicy(Policy):
     @staticmethod
     def _standard_featurizer(max_history: Optional[int] = None) -> TrackerFeaturizer:
         if max_history is None:
-            return FullDialogueTrackerFeaturizer(LabelTokenizerSingleStateFeaturizer())
+            return FullDialogueTrackerFeaturizer(BOWSingleStateFeaturizer())
```

Comment:
let's get rid of it and use `MaxHistoryTrackerFeaturizer` with infinite `max_history`